docs/variants/pc_engines/firmware-update.md

```diff
@@ -0,0 +1,51 @@
+# Legacy -> UEFI Update
+
+This document describes the process of updating the platform's legacy SeaBIOS
+to Dasharo firmware based on EDK II.
+
+## OS compatibility
+
+Switching from BIOS to UEFI-based firmware might cause compatibility issues if
+you already had an operating system installed on your platform. Some OSs can
+handle the switch without any issues, while others may need to be reinstalled.
+We have tested the update on several operating systems. The results are
+presented in the table below:
+
+### Ubuntu
+
+Can be booted from UEFI.
+
+### OPNSense
+
+Can be booted from UEFI.
+
+### pfSense
+
+Requires OS reinstall.
+
+### OpenWrt
+
+Depending on installed image, can be booted from UEFI or has to be reinstalled.
```

Comment:
Maybe OpenWRT can also be in a table, with a [footnote](https://squidfunk.github.io/mkdocs-material/reference/footnotes/) with additional information

Reply:
Done 83dfe802f485ddf1c5e0c5aca492460413373ea1